packages/block-library/src/navigation-menu/edit.js

```diff
@@ -23,6 +23,14 @@ import useBlockNavigator from './use-block-navigator';
 
 function NavigationMenu( {
 	attributes,
+<<<<<<< HEAD
+=======
+	clientId,
+<<<<<<< HEAD
+>>>>>>> horizontal option for the mover, missing icons, broken hover
+=======
+	isSelected,
+>>>>>>> positioned the mover to the middle left
```

Comment:
You have merge conflict markers here 🙈

Reply:
ouch, I botched up something apparently ... hasty merge